params/build_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2017 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// Package params contains build information of executable usually provided by
+// automated build systems like Travis. Default values are populated if not overriden by build system
+package params
```

Comment:
This package name is misleading, maybe "metadata"? Because it contains build info, version info, licenses etc etc. What kind of parameters are these ? Or what's bad with separate package for everything like "build" , "legal", "version" ?